### PR TITLE
Created two additional python test files for the USR2 and USR3 onboar…

### PR DESCRIPTION
**Pull request by iliescua**
…d LEDs

# Date
12/14/2020
Initial PR Date:
12/14/2020
Due Date: 

# Reviewers
@rockboynton 

# Description
Created two more python test files for LED USR2 and USR3 so now all the onboard LEDs are accessible. 

# Project Issue
[Create Test "user programs" #9](https://github.com/Senior-Design-0x07/the-hobby-hub/issues/9)

# Test Results
Verified because all of the LEDs flash as expected when their subsequent file is run.

### CODE REVIEW

File: files/test_programs/light2.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Adafruit_BBIO.GPIO as GPIO
+import time
+
+USR1_LED = "USR2"
```
Comment (rockboynton) [inline files/test_programs/light2.py]: Fix naming

File: files/test_programs/light3.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Adafruit_BBIO.GPIO as GPIO
+import time
+
+USR1_LED = "USR3"
```
Comment (rockboynton) [inline files/test_programs/light3.py]: Fix naming

**Review by rockboynton — CHANGES_REQUESTED** [review]

**Comment by iliescua** [comment]
Fixed it

**Comment by Ivendrakeo** [comment]
Looks good to me. Assuming its good for you, rock merge it.

**Comment by rockboynton** [comment]
😊

**Pull request merged**